roles/pulp_webserver/templates/pulp-vhost.conf.j2

```diff
@@ -29,3 +30,50 @@ Define pulp-api {{ pulp_api_bind }}
   ProxyPassReverse / http://${pulp-api}/
 
 </VirtualHost>
+{% else %}
+<VirtualHost *:80>
+  ServerName {{ pulp_webserver_httpd_servername }}
+  Redirect permanent / https://{{ pulp_webserver_httpd_servername }}/
+</VirtualHost>
+
+<VirtualHost *:443>
+  ServerName {{ pulp_webserver_httpd_servername }}
+
+  ## TLS Configuration
+  SSLEngine On
+  SSLCertificateFile {{ pulp_webserver_tls_folder }}/pulp_webserver.crt
+  SSLCertificateKeyFile {{ pulp_webserver_tls_folder }}/pulp_webserver.key
+  SSLProtocol TLSv1.2
+  SSLCipherSuite ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-SHA384:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256
```

Comment:
Do we really want to specify these? If we do not, will the system use some (sane/secure) distro default or system-wide crypto policy? If so, that would be much preferable.

Reply:
That part was done by @Spredzy and i have no idea. So any expert on the matter, please weigh in.